backend/src/events/round/functions/endRound.ts

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
Último caso de importação do tipo `Player`

```suggestion
import Player from "../../../interfaces/Player";

```